DcCore/DcCore/DC/events.swift

```diff
@@ -56,118 +56,105 @@ public class DcEventHandler {
-            }
+            
+            NotificationCenter.default.post(name: Event.importExportProgress, object: nil, userInfo: [
+                "progress": Int(data1),
+                "error": Int(data1) == 0,
+                "done": Int(data1) == 1000,
+                "errorMessage": self.dcAccounts.get(id: accountId).lastErrorString,
+            ])
         case DC_EVENT_MSGS_CHANGED:
             guard accountId == dcAccounts.getSelected().id else { return }
 
             logger.info("📡[\(accountId)] msgs changed: \(data1), \(data2)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.messagesChanged, object: nil, userInfo: [
-                    "message_id": Int(data2),
-                    "chat_id": Int(data1),
-                ])
-            }
+            
+            NotificationCenter.default.post(name: Event.messagesChanged, object: nil, userInfo: [
+                "message_id": Int(data2),
+                "chat_id": Int(data1),
+            ])
 
         case DC_EVENT_REACTIONS_CHANGED, DC_EVENT_MSG_READ, DC_EVENT_MSG_DELIVERED, DC_EVENT_MSG_FAILED:
             guard accountId == dcAccounts.getSelected().id else { return }
 
             logger.info("📡[\(accountId)] msgs reaction/read/delivered/failed: \(data1), \(data2)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.messageReadDeliveredFailedReaction, object: nil, userInfo: [
-                    "message_id": Int(data2),
-                    "chat_id": Int(data1),
-                ])
-            }
+
+            NotificationCenter.default.post(name: Event.messageReadDeliveredFailedReaction, object: nil, userInfo: [
+                "message_id": Int(data2),
+                "chat_id": Int(data1),
+            ])
 
         case DC_EVENT_MSGS_NOTICED:
             if accountId != dcAccounts.getSelected().id {
                 return
             }
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.messagesNoticed, object: nil, userInfo: [
-                    "chat_id": Int(data1),
-                ])
-            }
+            
+            NotificationCenter.default.post(name: Event.messagesNoticed, object: nil, userInfo: [
+                "chat_id": Int(data1),
+            ])
 
         case DC_EVENT_CHAT_MODIFIED:
             if accountId != dcAccounts.getSelected().id {
                 return
             }
             logger.info("📡[\(accountId)] chat modified: \(data1)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.chatModified, object: nil, userInfo: [
-                    "chat_id": Int(data1),
-                ])
-            }
+
+            NotificationCenter.default.post(name: Event.chatModified, object: nil, userInfo: [
+                "chat_id": Int(data1),
+            ])
         case DC_EVENT_CHAT_EPHEMERAL_TIMER_MODIFIED:
```

Comment:
```suggestion
            ])

        case DC_EVENT_CHAT_EPHEMERAL_TIMER_MODIFIED:
```

(just normalising whitespace)